src/Sulu/Bundle/PageBundle/Resources/phpcr-migrations/Version202404051700.php

```diff
@@ -0,0 +1,190 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\PageBundle;
+
+use PHPCR\Migrations\VersionInterface;
+use PHPCR\NodeInterface;
+use PHPCR\SessionInterface;
+use Sulu\Component\Content\Document\Subscriber\ShadowCopyPropertiesSubscriber;
+use Sulu\Component\Webspace\Webspace;
+use Symfony\Component\DependencyInjection\ContainerAwareInterface;
```

Comment:
This is deprecated and need to be replace on 2.6 with:

```suggestion
use PHPCR\PhpcrMigrationsBundle\ContainerAwareInterface;
```